content/docs/07-reference/01-ai-sdk-core/25-json-schema.mdx

```diff
@@ -14,7 +14,7 @@ You can use it to [generate structured data](/docs/ai-sdk-core/generating-struct
 (e.g. when using OpenAPI definitions) or for using other validation libraries.
 
 ```ts
-import { jsonSchema } from 'ai';
+import { jsonSchema } from '@ai-sdk/provider-utils';
```

Comment:
we still need to re-export from `ai` for end users of the `ai` library

Reply:
yep I did that, just thought i'd show it here you can do from ai-sdk/provider-utils as well